proposals/lip-0050.md

```diff
@@ -190,21 +291,17 @@ This module does not expose any functions.
 
 This function provides the legacy address and balance of the corresponding legacy accounts.
 
-##### Parameters
-
-* inputParameter1: `publicKey`, an Ed25519 public key
-
 ##### Returns
 
 Either an object with the properties `legacyAddress` and `balance`, where `legacyAddress` is the legacy address for `publicKey` and `balance` is the balance of the corresponding legacy account, or `undefined`.
```

Comment:
In this version of the function you don't return `undefined` anymore. Say that you return empty object if there is no legacy account.

Reply:
Changed that, also taking into account the comment by Shu.